crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -566,7 +543,7 @@ impl AlgorithmUpdaterV1 {
         let projection_portion: u128 = self
             .unrecorded_blocks
             .iter()
-            .map(|(_, &bytes)| (bytes as u128))
+            .map(|(_, &bytes)| bytes as u128)
```

Comment:
Again unrelated but it would be nice to use and explicit infallible conversion: `u128::from`.